### PR TITLE
[DO-NOT-MERGE] Document adding OpenShift Version in the Report

### PR DESCRIPTION
**Pull request by baijum**
The chart-verifier tool adds the OpenShift version where chart-testing has run in the report.

### CODE REVIEW

File: docs/openshift-version-in-the-report.md
Comment (HarshCasper) [inline docs/openshift-version-in-the-report.md:6]: Please hard wrap this into one single line.

File: docs/openshift-version-in-the-report.md
Comment (HarshCasper) [inline docs/openshift-version-in-the-report.md:4]: Remove the extra blank space before `The information`

File: docs/openshift-version-in-the-report.md
Comment (HarshCasper) [inline docs/openshift-version-in-the-report.md:19]: Please hard wrap this into one single line.

File: docs/openshift-version-in-the-report.md
Comment (HarshCasper) [inline docs/openshift-version-in-the-report.md:56]: Please hard wrap this into one single line.

File: docs/openshift-version-in-the-report.md
Comment (HarshCasper) [inline docs/openshift-version-in-the-report.md:61]: Please hard wrap this into one single line.

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md:4]: I used to use two spaces.
Here is some of the reasons: https://stevelosh.com/blog/2012/10/why-i-two-space/ (Not my blog post)

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md:6]: Text wrapping at 80 columns is good for PR review (diff) and writing in a text editor from a terminal.

File: docs/openshift-version-in-the-report.md
Comment (Srivaralakshmi) [inline docs/openshift-version-in-the-report.md:54]: Suggest: You can replace the values for the name and namespace of the ServiceAccount in the

Also, make this as a Note and the 1st bulleted point in the Note.

File: docs/openshift-version-in-the-report.md
Comment (Srivaralakshmi) [inline docs/openshift-version-in-the-report.md:55]: Suggest: preceding example (or) previous example.

Comment (Srivaralakshmi) [inline docs/openshift-version-in-the-report.md:55]: Suggest: Make this the 2nd bulleted point inside the Note: "Instead of"

File: docs/openshift-version-in-the-report.md
Comment (Srivaralakshmi) [inline docs/openshift-version-in-the-report.md:58]: Suggest: If configuring the role as mentioned in the previous examples is not ...

File: docs/openshift-version-in-the-report.md
Comment (Srivaralakshmi) [inline docs/openshift-version-in-the-report.md:60]: Suggest: For example, you can use the `--openshift-version` flag to specify the OpenShift version in the **chart-verifier** CLI tool:

Comment (Srivaralakshmi) [inline docs/openshift-version-in-the-report.md:60]: Suggest: Delete "Here is an example:"

File: docs/openshift-version-in-the-report.md
Comment (Srivaralakshmi) [inline docs/openshift-version-in-the-report.md:67]: Suggest: Put it as a Note.

File: docs/openshift-version-in-the-report.md
Comment (Srivaralakshmi) [inline docs/openshift-version-in-the-report.md:69]: Suggest: ## Example

**Review by Srivaralakshmi — COMMENTED** [review]
@baijum Left some comments. PTAL. Thanks!

**Comment by Srivaralakshmi** [comment]
@baijum Left some comments. PTAL. Thanks!

File: docs/openshift-version-in-the-report.md
Comment (mmulholla) [reply docs/openshift-version-in-the-report.md:6]: What is the benefit of a single line?  I find multi-line version is easier to work with.

File: docs/openshift-version-in-the-report.md
Comment (mmulholla) [reply docs/openshift-version-in-the-report.md:4]: What is the benefit of a single space?

File: README.md
Comment (baijum) [reply README.md:24]: That text is not part of this PR. Let's use separate PR to correct it.

File: README.md
Comment (baijum) [reply README.md:22]: I have already placed it under the "docs" folder. The link should work once the PR is merged.

File: README.md
Comment (baijum) [reply README.md:19]: That text is not part of this PR. Let's use separate PR to correct it.

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md:6]: Introducing a sub-heading will make the paragraph following the example also part of the sub-section. So, I am going to make it a bold text like this:

**Example**

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md:16]: This sentence doesn't make sense. It is not about whether the user has logged in or not.

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md:17]: Why do we need a definite article in front of `access`?

Comment (baijum) [reply docs/openshift-version-in-the-report.md:17]: Please see the example below. Here is a relevant snippet from the example:
```
    resources:
      - 'clusteroperators'
    verbs:
      - 'get'
```

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md:78]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (baijum) [reply docs/openshift-version-in-the-report.md:4]: Done. 67fccc10087c7c67766d77b2b9384bd1762538c4

File: docs/openshift-version-in-the-report.md
Comment (Srivaralakshmi) [inline docs/openshift-version-in-the-report.md:56]: Suggest: Replace Notes with Note

**Review by Srivaralakshmi — COMMENTED** [review]
@baijum just a minor comment. Otherwise LGTM.

**Comment by baijum** [comment]
Let's hold this PR until discussion in this PR is completed: #133

File: docs/openshift-version-in-the-report.md
Comment (Srivaralakshmi) [reply docs/openshift-version-in-the-report.md]: RH writing guidelines: Hard wrapping makes the code nice and tidy, as there are some editors that do not wrap lines when you open a new file. It allows you to concentrate on the structure of your document. Good indentation and good wrapping make it more readable by most of users. Please do take it as a habit :).

File: docs/openshift-version-in-the-report.md
Comment (Srivaralakshmi) [reply docs/openshift-version-in-the-report.md:4]: RH style guide and writing guidelines says so. Also, the majority of style guides now prescribe the use of a single space after terminal punctuation in final written works and publications. We should update our writing to reflect current tools and an extra space between sentences creates a river of white in documents, diminishing the impact.

Please read: [https://www.cultofpedagogy.com/two-spaces-after-period/](https://www.cultofpedagogy.com/two-spaces-after-period/) and [https://www.instructionalsolutions.com/blog/one-space-vs-two-after-period](https://www.instructionalsolutions.com/blog/one-space-vs-two-after-period). Thanks!

File: README.md
Comment (Srivaralakshmi) [reply README.md:24]: Sure @baijum! Just ensure that all such comments that are left unresolved in this PR are implemented in the separate PR. Thanks! (Global comment)

Comment (baijum) [reply README.md:24]: I have sent a separate PR: #138

File: docs/openshift-version-in-the-report.md
Comment (mmulholla) [reply docs/openshift-version-in-the-report.md:4]: Is there a tool to format a file according to the RH Sytle guide?

**Comment by baijum** [comment]
This documentation is not required as there is no need to create a cluster role.

Ref. #133

**Pull request closed without merging**
